PythonScript/src/MenuManager.cpp

```diff
@@ -837,7 +837,7 @@ idx_t MenuManager::findPluginCommand(const TCHAR *pluginName, const TCHAR *menuO
 	{
 		HMENU hPluginMenu = (HMENU)::SendMessage(m_hNotepad, NPPM_GETMENUHANDLE, 0, 0);
 
-		size_t iMenuItems = (size_t)GetMenuItemCount(hPluginMenu);
+		int iMenuItems = GetMenuItemCount(hPluginMenu);
 		TCHAR strBuffer[500]{};
 		for ( idx_t i = 0; i < iMenuItems; ++i )

```

Comment:
As GetMenuItemCount() could return -1 in case of an error the for loop with unsigned idx_t could create a problem.